src/player/mod.rs

```diff
@@ -1,17 +1,49 @@
-use std::net::{SocketAddr};
+use std::net::{SocketAddr, IpAddr, UdpSocket};
+use std::io;
 
 #[derive(Debug, Clone)]
 pub struct Player {
     pub model: String,
     pub address: SocketAddr,
     pub number: u8,
+    pub linked: bool,
 }
 
-// impl Player {
-//     pub fn verify(&self) -> Result<(), Error> {
-//        Err()
-//     }
-// }
+impl Player {
+    pub fn ip(&self) -> IpAddr {
+        return self.address.ip()
+    }
+
+    pub fn link(socket: &UdpSocket) -> io::Result<usize> {
+        socket.set_broadcast(true).expect("set_broadcast call failed");
+
+        // This should be sent to broadcast
+        let buffer = [
+            0x51,0x73,0x70,0x74,0x31,0x57,
```

Comment:
Figure out a nice way to construct UDP packages. Some of this data is kinda stateful / static in here.

Like computer name (seems to be treated like a visual name in the CDJs display window).